libs/search-widget/src/core/stores/search.store.ts

```diff
@@ -98,10 +98,14 @@ export const searchResults = searchState.writer<
 >(
   (state) => state.results,
   (state, params) => {
-    CACHE = undefined;
+    const sortedResults = getSortedResults(Object.values(params.results.resources || {}));
     return {
       ...state,
-      results: params.append ? appendResults(state.results, params.results) : params.results,
+      results: {
+        ...params.results,
+        resources: undefined,
```

Comment:
not needed

Reply:
If we don't do that, we have the resources duplicated between `resources` and `resultList` property